.github/workflows/ci.yaml

```diff
@@ -3,6 +3,8 @@ on:
   pull_request:
     branches:
       - main
+    paths-ignore:
```

Comment:
@mikemhenry can we put a comment here about why this entry exists?  i.e. I think this one is something like "if only docs/ touched it doesn't require the unit tests part of the matrix"

Reply:
Done in 51ea98ef63768b3d46fb627def6387e9b5c11943